From 0ec8b2e1c07b8ee99dd0fe9a9138a0d56e0ed040 Mon Sep 17 00:00:00 2001
From: Adam Ford <aford173@gmail.com>
Date: Sun, 12 Apr 2020 08:14:07 -0500
Subject: [PATCH 12/17] arm64: dts: logicpd-imx8mm-kit: Expliticly set load 
 capacitance

With the (mostly) backported driver for the pcf85263/pcf85363,
this patch explitcly sets the capacitance to 12.5pF.  This is
the driver default, but in the event the default changes, this
value should not for this board.

Fixes: PHOENIX8-23

Signed-off-by: Adam Ford <aford173@gmail.com>
---
 arch/arm64/boot/dts/freescale/beacon-imx8mm-som.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/freescale/beacon-imx8mm-som.dtsi b/arch/arm64/boot/dts/freescale/beacon-imx8mm-som.dtsi
index 3a0525d51172..b08b7d7fa46f 100644
--- a/arch/arm64/boot/dts/freescale/beacon-imx8mm-som.dtsi
+++ b/arch/arm64/boot/dts/freescale/beacon-imx8mm-som.dtsi
@@ -217,6 +217,7 @@
 	rtc@51 {
 		compatible = "nxp,pcf85263";
 		reg = <0x51>;
+		quartz-load-femtofarads = <12500>;
 	};
 };
 
-- 
2.17.1

